tests/ui-toml/item_name_repetitions/threshold0/item_name_repetitions.rs

```diff
@@ -0,0 +1,3 @@
+struct Data {}
```

Comment:
Can we also add `enum Actions {}` here? This PR also deletes the enum_variant_threshold0 uitest from the other PR, so right now this doesn't test empty enums anymore.

Reply:
ah yes, I hadn't initially added because it always failed (because of the ice fixed in the other pr). During the rebase I must have missed it